LICENSE-MIT

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015-2018 Sean Griffin
```

Comment:
I'm not a lawyer but I would like to change that to 2015-2020 The diesel project or something like that?

Reply:
Yep, it makes sense.